src/book/03-guides/02-tools/03-webots.mdx

```diff
@@ -76,51 +76,68 @@ sudo apt-get install cmake-curses-gui libprotobuf-dev protobuf-compiler libeigen
 
 ## Install Webots
 
-1. Open a terminal and swap to your home directory (note that Webots should be cloned in your home directory). Then clone the repository and change into the cloned directory:
+1. Open a terminal, clone the NUWebots repository and change into the cloned directory:
 
    ```bash
    cd ~
```

Comment:
Doesn't need to be in home directory any more but not sure if it's worth mentioning this